### PR TITLE
[SSO] Azure+  OIDC guide

### PR DESCRIPTION
**Pull request by SBrandeis**
@Kakulukian @glegendre01 

Screenshots are in this Hub PR: https://huggingface.co/datasets/huggingface/documentation-images/commit/b134c56c2d4748be0a161ed13211407228f34553

Reusing the structure from other guides

### CODE REVIEW

File: docs/hub/security-sso-azure-oidc.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# How to configure OIDC SSO with Azure
+
+In this guide, we will use Azure as the SSO provider and with the Open ID Connect (OIDC) protocol as our preferred identity protocol. 
+
+We currently support SP-initiated and IdP-initiated authentication. User provisioning is not yet supported at this time.
```
Comment (Kakulukian) [inline docs/hub/security-sso-azure-oidc.md:5]:
```suggestion
```

IdP-initiated authentication doesn't exist for OIDC so we can remove this part

**Review by Kakulukian — APPROVED** [review]
Didn't check images, preview seems down cc @mishig25

File: docs/hub/security-sso-azure-oidc.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# How to configure OIDC SSO with Azure
+
+In this guide, we will use Azure as the SSO provider and with the Open ID Connect (OIDC) protocol as our preferred identity protocol.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:3]:
```suggestion
This guide will use Azure as the SSO provider and the Open ID Connect (OIDC) protocol as our preferred identity protocol. 
```

File: docs/hub/security-sso-azure-oidc.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+<Tip warning={true}>
+	This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
+
+
+### Step 1: Create a new application in your Identity Provider
+
+Open a new tab/window in your browser and sign in to the Azure portal of your organization.
+
+Navigate to the Microsoft Entra ID admin center, and click on "Enterprise applications"
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:15]:
```suggestion
Navigate to the Microsoft Entra ID admin center and click on "Enterprise applications"
```

File: docs/hub/security-sso-azure-oidc.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-1.png"/>
+</div>
+
+You'll be redirected to this page, click on "New application" at the top, then "Create your own application".
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:21]:
```suggestion
You'll be redirected to this page. Then click "New application" at the top and "Create your own application".
```

File: docs/hub/security-sso-azure-oidc.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-2.png"/>
+</div>
+
+Input a name for your application (for example, Hugging Face SSO), then select "Register an application to integrate with Microsoft Entra ID (App you're developping)".
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:27]:
```suggestion
Input a name for your application (for example, Hugging Face SSO), then select "Register an application to integrate with Microsoft Entra ID (App you're developing)".
```

File: docs/hub/security-sso-azure-oidc.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/sso-navigation-settings-dark.png"/>
+</div>
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/sso-settings.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/sso/sso-settings-dark.png"/>
+</div>
+
+
+Copy the "Redirection URI" from the organization's settings on Hugging Face, and paste it in the "Redirect URI" field on Azure Entra ID. Make sure you selected "Web" in the dropdown menu.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:48]:
```suggestion
Copy the "Redirection URI" from the organization's settings on Hugging Face and paste it into the "Redirect URI" field on Azure Entra ID. Make sure you select "Web" in the dropdown menu.
```

File: docs/hub/security-sso-azure-oidc.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+</div>
+
+In the application overview, click on "Single sign-on", then "Go to application"
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-6.png"/>
+</div>
+
+In the OIDC app overview, you will fnd a copiable field named "Application (client) ID".
+Copy that ID to your clipboard and paste it in the "Client ID" field on Huggingface.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:77]:
```suggestion
In the OIDC app overview, you will find a copiable field named "Application (client) ID".
Copy that ID to your clipboard and paste it into the "Client ID" field on Huggingface.
```

File: docs/hub/security-sso-azure-oidc.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-7.png"/>
+</div>
+
+Next, in Microsoft Entra, click on "Endpoints" in the top menu.
+Copy the value in the "OpenID connect metadata document" field, and paste it in the "Issue URL" field in Hugging Face.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:84]:
```suggestion
Next, click "Endpoints" in the top menu in Microsoft Entra.
Copy the value in the "OpenID connect metadata document" field and paste it into the "Issue URL" field in Hugging Face.
```

File: docs/hub/security-sso-azure-oidc.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-8.png"/>
+</div>
+
+Back in Microsoft Entra, navigate to "Certificates & secrets", and generate a new secret by clicking "New client secret".
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-9.png"/>
+</div>
+
+Once you have created the secret, copy the secret value and paste it in the "Client secret" field on Hugging Face.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:96]:
```suggestion
Once you have created the secret, copy the secret value and paste it into the "Client secret" field on Hugging Face.
```

File: docs/hub/security-sso-azure-oidc.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/b134c56c2d4748be0a161ed13211407228f34553/hub/sso/sso-azure-oidc-guide-10.png"/>
+</div>
+
+
+You can now click on "Update and Test OIDC configuration" to save the settings.
```
Comment (osanseviero) [inline docs/hub/security-sso-azure-oidc.md:103]:
```suggestion
You can now click "Update and Test OIDC configuration" to save the settings.
```

**Review by osanseviero — APPROVED** [review]
Nice!

**Pull request merged**